adapter/api/src/main/java/org/apache/rocketmq/eventbridge/adapter/api/controller/ApiDestinationController.java

```diff
@@ -62,83 +64,137 @@ public class ApiDestinationController {
 
     @WebLog
     @PostMapping("createApiDestination")
-    public CreateApiDestinationResponse createApiDestination(@RequestBody CreateApiDestinationRequest createApiDestinationRequest) {
-        final Set<ConstraintViolation<CreateApiDestinationRequest>> validate = validator.validate(createApiDestinationRequest);
-        List<String> errMessage = validate.stream().map(ConstraintViolation::getMessage).collect(Collectors.toList());
-        if (!CollectionUtils.isEmpty(errMessage)) {
-            return new CreateApiDestinationResponse(null).parameterCheckFailRes(errMessage.toString());
-        }
-        ApiDestinationDTO apiDestinationDTO = getEventApiDestination(createApiDestinationRequest.getHttpApiParameters(), createApiDestinationRequest.getDescription(), createApiDestinationRequest.getConnectionName(), createApiDestinationRequest.getInvocationRateLimitPerSecond(), createApiDestinationRequest.getApiDestinationName(), accountAPI);
-        return new CreateApiDestinationResponse(apiDestinationService.createApiDestination(apiDestinationDTO)).success();
+    public Mono<CreateApiDestinationResponse> createApiDestination(
+        @RequestBody CreateApiDestinationRequest createApiDestinationRequest) {
+        return Mono.subscriberContext()
```

Comment:
Mono.subscriberContext seems to be deprecated according to this [issue](https://github.com/reactor/reactor-core/issues/2572)